modules/images-other-jenkins-agent-memory.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-memory_{context}"]
+= Jenkins agent memory requirements
+
+A JVM is used in all Jenkins agents to host the Jenkins JNLP agent, as well as
+to run any Java applications such as `javac`, Maven, or Gradle.
+
+For memory efficiency, by default the Jenkins image dynamically uses a 32-bit
```

Comment:
This section should be removed, since OPENSHIFT_JENKINS_JVM_ARCH can't be overrided with 4.x images.

Reply:
Commented out for now. Should the whole module be removed? Thanks!